create_package.sh

```diff
@@ -25,7 +25,7 @@ mkdir -vp /ovms_release/lib/custom_nodes
 if [ -f /openvino_tokenizers/build/src/libopenvino_tokenizers.so ]; then cp -v /openvino_tokenizers/build/src/libopenvino_tokenizers.so /ovms_release/lib/ ; fi
 if [ -f /openvino_tokenizers/build/src/libcore_tokenizers.so ]; then cp -v /openvino_tokenizers/build/src/libcore_tokenizers.so /ovms_release/lib/ ; fi
 
-find /ovms/bazel-out/k8-*/bin -iname '*.so*' ! -type d ! -name "*params" ! -name "lib_node_*" ! -path "*test_python_binding*" ! -name "*libpython*" -exec cp -v {} /ovms_release/lib/ \;
```

Comment:
W should specify concrete files to be copied or the regex what to copy should be much more strict. What if another files occurs with slightly different name? We could either copy *openvino*.so* & *opencv*.so* and then specify few additional files, or specify all of them.